docs/source/index.md

```diff
@@ -20,113 +20,183 @@ ExecuTorch provides support for:
 
 * **Strong Model Support** LLMs (Large Language Models),
   CV (Computer Vision), ASR (Automatic Speech Recognition), TTS (Text To Speech)
-* **All Major Platforms** Android, Mac, Linux, Windows
+* **All Major Platforms** Android, iOS, macOS, Linux, Windows, Embedded
 * **Rich Acceleration Support** Apple, Arm, Cadence, MediaTek, NXP, OpenVino, Qualcomm, Vulkan, XNNPACK
 
 ### Documentation Navigation
-#### Introduction
+
+#### Getting Started
 - [Overview](intro-overview)
 - [How it Works](intro-how-it-works)
-- [Getting Started with Architecture](getting-started-architecture)
-- [Concepts](concepts)
-#### Usage
-- [Getting Started](getting-started)
-- [Using Executorch Export](using-executorch-export)
-- [Using Executorch on Android](using-executorch-android)
-- [Using Executorch on iOS](using-executorch-ios)
-- [Using Executorch with C++](using-executorch-cpp)
-- [Runtime Integration](using-executorch-runtime-integration)
-- [Troubleshooting](using-executorch-troubleshooting)
+- [Architecture & Components](getting-started-architecture)
+- [Core Concepts](concepts)
 - [Building from Source](using-executorch-building-from-source)
-- [Quantization](quantization-overview)
-- [FAQs](using-executorch-faqs)
-#### Examples
-- [Android Demo Apps](https://github.com/meta-pytorch/executorch-examples/tree/main/dl3/android/DeepLabV3Demo#executorch-android-demo-app)
-- [iOS Demo Apps](https://github.com/meta-pytorch/executorch-examples/tree/main/mv3/apple/ExecuTorchDemo)
-- [Hugging Face Models](https://github.com/huggingface/optimum-executorch/blob/main/README.md)
-#### Backends
-- [Overview](backends-overview)
-- [XNNPACK](backends-xnnpack)
-- [Core ML](backends-coreml)
-- [MPS](backends-mps)
-- [Vulkan](backends-vulkan)
-- [ARM Ethos-U](backends-arm-ethos-u)
-- [ARM VGF](backends-arm-vgf)
-- [Qualcomm](backends-qualcomm)
-- [MediaTek](backends-mediatek)
-- [Cadence](backends-cadence)
-- [OpenVINO](build-run-openvino)
-- [NXP](backend-nxp)
-#### Developer Tools
-- [Overview](devtools-overview)
-- [Bundled IO](bundled-io)
-- [ETRecord](etrecord)
-- [ETDump](etdump)
-- [Runtime Profiling](runtime-profiling)
-- [Model Debugging](model-debugging)
+
+#### Fundamentals
+- [Model Export & Lowering](using-executorch-export)
+- [Runtime Architecture](runtime-overview)
+- [Memory Management](compiler-memory-planning)
+- [Performance Concepts](runtime-profiling)
+
+#### Edge Platforms
+##### Android
+- [Quick Start & Integration](using-executorch-android)
+- ```{dropdown} Available Backends
+  :open:
+    - [XNNPACK (CPU)](backends-xnnpack)
+    - [Vulkan (GPU)](backends-vulkan)
+    - [Qualcomm AI Engine (NPU)](backends-qualcomm)
+    - [MediaTek (NPU)](backends-mediatek)
+    - [Samsung Exynos (NPU)](backends-samsung-exynos)
+  ```
```

Comment:
Include Arm VGF

https://docs.pytorch.org/executorch/1.0/backends-arm-vgf.html

Reply:
Where shall we include https://docs.pytorch.org/executorch/1.0/tutorial-arm-vgf.html